toolchain/sem_ir/import_ir.h

```diff
@@ -30,6 +30,10 @@ struct ImportIRInst : public Printable<ImportIRInst> {
     out << ir_id << ":" << inst_id;
   }
 
+  auto operator==(const ImportIRInst& rhs) const -> bool {
+    return ir_id == rhs.ir_id && inst_id == rhs.inst_id;
+  }
```

Comment:
```suggestion
  auto operator==(const ImportIRInst& rhs) const -> bool = default;
```

Reply:
```
In file included from toolchain/sem_ir/name.cpp:8:
In file included from ./toolchain/sem_ir/file.h:20:
./toolchain/sem_ir/import_ir.h:33:8: error: explicitly defaulted equality comparison operator is implicitly deleted [-Werror,-Wdefaulted-function-deleted]
  auto operator==(const ImportIRInst& rhs) const -> bool = default;
       ^
./toolchain/sem_ir/import_ir.h:28:30: note: defaulted 'operator==' is implicitly deleted because there is no viable three-way comparison function for base class 'Printable<Carbon::SemIR::ImportIRInst>'
struct ImportIRInst : public Printable<ImportIRInst> {
```

Reply:
Adding an explicit operator== to Printable fixes _this_, but makes all other children comparable too. I'm not sure that's what we want. Will follow up separately.